### ISSUE TITLE
Add how to uninstall

### ISSUE THREAD

**Issue by lwcorp** [GAM-team/GAM#1364]
**Is your feature request related to a problem? Please describe.**
GAM's Google Group mentions all it takes to uninstall is:
1. **Windows users that installed via the MSI file**, use the official uninstallation
1. **Anyone else**, simply delete the GAM folder

**Describe the solution you'd like**
1. Add a clear and concise description of uninstallation to https://github.com/jay0lee/GAM/wiki
1. If needed, add what else needs to be done other than deleting the GAM folder

**Describe alternatives you've considered**
At least add a link to https://groups.google.com/g/google-apps-manager/c/N8XH-aRv7eE/m/tkvHuZSoCQAJ which explains the above.

**Comment by lwcorp** [comment]
Alright, I've added [it to the wiki](https://github.com/jay0lee/GAM/wiki#uninstall-gam)! But I'd appreciate further easy technical instructions to add to the currently very basic `cloud uninstallation` part, and how to remove possibly leftover environment variables in Windows.

**Comment by kinafu** [comment]
[GAM 6.5.8](https://github.com/GAM-team/GAM/releases/tag/v6.58) uninstaller component in the installer for windows does not seem to work for me.
After I click on uninstall, a new setup window opens and on clicking "Continue" nothing happens.

**Comment by jay0lee** [comment]
closing as we now have uninstall instructions and MSI uninstall of GAM7 worked for me when tested recently. If you still see issues please start a discussion at https://git.io/gam-group

**Comment by lwcorp** [comment]
"Now" is actually 2021 [when I added this to the wiki myself](https://github.com/GAM-team/GAM/wiki/Home/_compare/3044e9c03f8fd7ba1cd3a065f82a5a78650cbfe6...20e2b97230589f6b867a52a59d6c61211842cf97) after getting no reply here. I've just documented what was in the Google Group.
This is why it doesn't mention how to uninstall the Windows' ZIP version. Perhaps like Linux/Mac but who knows for sure.
All I wanted was official confirmation to that Google Group information.
